pom.xml

```diff
@@ -93,11 +95,30 @@
                 </executions>
             </plugin>
 
-        </plugins>
-    </build>
-
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
 
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
+                            <mainClass>bowlingexercise3.BowlingApplication</mainClass>
+                        </manifest>
+                    </archive>
+                </configuration>
 
+            </plugin>
 
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
```

Comment:
You could try updating it to java 11?

Reply:
It is on 11. 

    <properties>
        <maven.compiler.source>11</maven.compiler.source>
        <maven.compiler.target>11</maven.compiler.target>
    </properties>